.github/workflows/automated-staging-test-run.yml

```diff
@@ -2,7 +2,7 @@ name: Automated Staging Test - Run integration tests
 
 on:
   schedule:
-    - cron: "0 2 * * 2-6"  # Tuesday to Saturday at 2am UTC - two hours after `automated-staging-test-submit` runs
+    - cron: "0 8 * * 2-6"  # Tuesday to Saturday at 2am CST (8am UTC) - two hours after `automated-staging-test-submit` runs
```

Comment:
Would it be helpful to have a [`WEBSITE_TIME_ZONE`](https://learn.microsoft.com/en-us/azure/azure-functions/functions-bindings-timer?tabs=python-v2%2Cisolated-process%2Cnodejs-v4&pivots=programming-language-typescript#ncrontab-time-zones) instead of converting from UTC to CST?

Reply:
What would be the advantage of using WEBSITE_TIME_ZONE?

Reply:
It would eliminate the need to know UTC is default and prevent potential future issues in forgetting to convert UTC to CST. I think it'd be less confusing as well since it's still technically UTC but in actuality we're using CST with extra conversion steps.

Reply:
The seems to be a feature of Azure functions but we're not using that here, just a github workflow with a cron schedule

Reply:
Yup, seems like there isn't a feature for that yet. I retract my suggestion